docs/OBOPURLDomain.md

```diff
@@ -1,11 +1,39 @@
 ---
 layout: doc
 id: OBOPURLDomain
-title: OBOPURLDomain
+title: OBO_PURL_Creation
 ---
+# Background #
 
-The [OBO ID Policy](https://obofoundry.org/id_policy.shtml) describes the general schema for which purls are then create, and what the format of individual purls are.
+Every ontology in OBO has a PURL, each version has a PURL, and every class also gets a PURL. The PURLs for ontologies can be found on individual ontology home pages.
+
+The [OBO ID Policy](http://www.obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html) describes the requirements and process for requesting a new PURL.
+
+For technical details of how the OBO PURL system works, and the format of individual purls, refer to https://github.com/OBOFoundry/purl.obolibrary.org/.
+
+# SOP for new PURL creation #
+
+This SOP is intended for use by the OBO Foundry Technical Working Group. If you are an ontology developer wanting to request a new PURL, please see the [PURL Request Guidelines](http://www.obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.htm). Savvy users may follow some of these steps to create their own ontology metadata files and then make a pull request, following the two week period of public comment.
+
+## Prerequisites ##
+1. Someone has requested a new PURL, following the [PURL Request Guidelines](http://www.obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.htm), i.e. they have already sent an email to [obo-discuss](mailto:obo-discuss@googlegroups.com) and created an issue on the [tracker](https://github.com/OBOFoundry/OBOFoundry.github.io/issues).
+
+## PURL request review process ##
+1. All members of the OBO community should respond to requests on obo-discuss as appropriate. This is particurly important if the new resource overlaps with an ontology you work on.
```

Comment:
typo

Reply:
I can't see this one

Reply:
There should be no closing `##`s

Reply:
also good practice to leave newlines either side